### DISCUSSION TITLE
Website Architecture Ideas

### DISCUSSION THREAD

**Discussion by txase** [pdxtroop150/troop150-website#3] [Ideas]
We need ideas for the overall architectural design of our website! Leave comments here with URLs to sites you like.

**Comment by txase** [comment]
https://pack214.com/

**Reply by readerkid** [reply]
We need to add a lost and found area

**Comment by Kolding3** [comment]
current troop website here: [https://troop150.org](https://troop150.org)

**Comment by sassyskiingpuppy19** [comment]
amazon.com. I chose this website because of the pictures and all the different options you can click on.

**Comment by sassyskiingpuppy19** [comment]
https://www.walmart.com, because of the setup. https://www.facebook.com, because you can comment on stuff. https://www.chewy.com, because you can click on different pets for stuff you need.  https://www.overstock.com, because of the formatting.

**Comment by txase** [comment]
t107pdx.com

**Comment by txase** [comment]
troop693.org
